src/about_us/index.js

```diff
@@ -11,9 +11,15 @@ import GetInTouch from 'main_app/components/GetInTouch'
 import PictureWall from 'main_app/components/PictureWall'
 import ContactModal from 'main_app/components/ContactModal'
 import Values from './components/Values'
+
+import usePageMetadata from 'utils/marketing/use_page_metadata'
 import TestimonialsBg from 'assets/about_us/testimonials/people_smiling.png'
 
 function AboutUs () {
+  usePageMetadata({
+    title: 'WeDevelop: About Us',
```

Comment:
Fijate que ya se le agrega un `WeDevelop - ` adelante del título que vos pongas. Como está ahora queda repetido en el título de la pestaña.
```suggestion
    title: 'About Us',
```